s3prl/downstream/speech_translation/README.md

```diff
@@ -0,0 +1,129 @@
+## ST: Speech Translation
+
+specified by the command `-d speech_translation`
+
+#### Prepare data
```

Comment:
I notice many external python packages that are unique for ST. Maybe consider list a `requirement.txt` or present some environment guidance at the `readme`

Reply:
Hi just for @bearhsiang 's information, we went from `requirement.txt` to `setup.py`, so you can update the package dependencies in `setup.py` at the root of the repo.

Reply:
Ok! I will add the dependence there.